cns/restserver/ipam.go

```diff
@@ -169,6 +210,23 @@ func (service *HTTPRestService) requestIPConfigsHandler(w http.ResponseWriter, r
 	if err != nil {
 		return
 	}
+
```

Comment:
Can we update the UT to verify these changes as well.

Reply:
There aren't any existing tests that use requestIPConfigsHandler.